RELEASING.md

```diff
@@ -1,8 +1,11 @@
 #### Releasing:
 1. Create a `fastlane/.env` file with GitHub API token (see `fastlane/SAMPLE.env`)
 2. Run `bundle exec fastlane bump`
-  1. Input new version number
-  2. Update changelog
-  3. A new branch and PR will automatically be created
-3. Merge PR when approved
-4. Make a tag and push, the rest will be performed automatically by CircleCI. If the automation fails, you can revert to manually calling `bundle exec fastlane deploy`.
+    1. Input new version number
+    2. Update changelog
+    3. A new branch and PR will automatically be created
+3. Generate new `RevenueCat-Swift.h` [for SPM](https://github.com/RevenueCat/purchases-ios/blob/main/docs/V4_API_Updates.md#objc--spm):
+    1. Run `bundle exec fastlane compile_autogenerated_header`
```

Comment:
I would even be happy to have this run on each deploy without having a separate lane. That way there's no thinking/possibility of missing it. We'll get tiny metadata updates, but I think that's fine.

Reply:
Then we'd also have to `git add && git commit && git push`, unless we also automate that, which I agree would be nice.

Reply:
I'm totally cool with that. I think we do that with the deploy script already for the version bumping.

Reply:
I was't familiar with the v4 release process cause I used the 3.x a few weeks ago, but now I am!
This is so much simpler 👍  and `bump` already does the right thing.